www/include/monitoring/recurrentDowntime/ajaxPeriods.php

```diff
@@ -36,30 +36,21 @@
  *
  */
 
-    header('Content-Type: application/json');
-    header('Cache-Control: no-cache');
+header('Content-Type: application/json');
+header('Cache-Control: no-cache');
 
-    require_once realpath(dirname(__FILE__) . "/../../../../config/centreon.config.php");
-    require_once _CENTREON_PATH_ . "/www/class/centreonDB.class.php";
-    require_once _CENTREON_PATH_ . "/www/class/centreonDowntime.class.php";
+require_once realpath(dirname(__FILE__) . "/../../../../config/centreon.config.php");
+require_once _CENTREON_PATH_ . "/www/class/centreonDB.class.php";
+require_once _CENTREON_PATH_ . "/www/class/centreonDowntime.class.php";
 
-    $pearDB = new CentreonDB();
-
-if (isset($_GET['dt_id'])) {
-    $id = $_GET['dt_id'];
-} else {
-    $id = 0;
-}
-
-    $path = _CENTREON_PATH_ . "www/include/monitoring/recurrentDowntime/";
+$downtimeId = filter_input(INPUT_GET, 'dt_id', FILTER_SANITIZE_NUMBER_INT);
```

Comment:
a downtime Id, shouldn't have letters in its param.
so you should use FILTER_VALIDATE_INT instead